dashboard/static/styles/main.styl

```diff
@@ -13,7 +15,7 @@ main {
 
 section {
     margin-top: 20px;
-    margin-bottom: 20px;
+    margin-bottom: @margin-top;
```

Comment:
I mean just to change the `margin-top` and `margin-bottom` declarations to a single short-hand `margin` declaration.